lang/en.js

```diff
@@ -359,7 +359,9 @@ export default {
     "exhibitions": "Exhibitions"
   },
   "actions": {
-    "download": "Download"
+    "download": "Download",
+    "viewAt": "View at",
```

Comment:
It can not be assumed that in all languages the provider name and link will go at the end of this text, so this needs a {provider} placeholder to take the link.